package.json

```
@@ -1,14 +1,12 @@
 {
-  "name": "safe-token",
+  "name": "@safe-global/safe-token",
   "version": "1.0.0",
   "description": "Repository for Safe token related contracts",
   "homepage": "https://www.safe.global",
-  "license": "GPL-3.0",
+  "license": "LGPL-3.0",
```

```suggestion
  "license": "LGPL-3.0-only",
```

The [`LGPL-3.0` identifier is deprecated](https://spdx.org/licenses/LGPL-3.0.html).